### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Ansper11**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# 499Project-Minecraft-C
\ No newline at end of file
+# 499Project-Minecraft-C
+
+Hello There are 
+
+Some ideas for this week. Thanks
```
Comment (eloiseespel) [inline README.md:5]: Yes I agree we should have ideas!

File: README.md [hunk 2]
```diff
@@ -1 +1,5 @@
-# 499Project-Minecraft-C
\ No newline at end of file
+# 499Project-Minecraft-C
+
+Hello There are
```
Comment (Nick-McGee) [inline README.md:3]: Just copy and paste your ideas over here.

Comment (Ansper11) [reply README.md:3]: AI and ML are based on computer science and programming, but the student in middle school may not know what python is, and in elementary school there are a lot of students using Block-base programming language like Scratch. Therefore, how to help middle school students to switch from Scratch to Python can be a good point in our project.
translate the block-based code to python code. E.g. make the python code as a password of a treasure box in the game and set the block code as password prompt. user can find the comparison table for block code and python code somewhere in the game.
 
Computer Vision System E.g. We can have some face recognition functions in the game like an alarm system in the yard, user can train an CV model and when a monster come to the yard the system will alert the user.
 
supervised/unsupervised learning
user can train a robot to do some specific job. E.g. train a robot to mining automatically. give a diamond to a robot and set as the prefer element to collect. And give a soil to the robot and set the soil as the element that must ignored. This is the concept of supervised learning
 
some programming games E.g. we can build some easy programming games in the game. For example, when user explore the game, there is a cave of the lost civilization, player should finish a programming game to open the door of the cave. (some easy games like sort an array or delete duplicate values in a list to help the player to be familiar with some programming knowledges)

Comment (yyht6300a) [reply README.md:3]: Nice idea.

**Pull request merged**